### ISSUE TITLE
Please add a function to remove multiple books at once from the library. 

### ISSUE THREAD

**Issue by Mewto7** [OptimiliaStudios/AquileReader#154]
I can not seem to be able to remove multiple books at one from the library. It is time-consuming to remove them one by one. A feature that lets me select multiple books at once and remove them would be nice. Thanks.

**Comment by oliverhr** [comment]
I agree with this feature, if you want at least for premium users ;)
